themes/bootstrap3/js/common.js

```diff
@@ -111,8 +111,8 @@ var VuFind = (function VuFind() {
   };
   var loading = function loading(text = null, extraClass = "") {
     let className = ("loading-spinner " + extraClass).trim();
-    let string = translate(text === null ? "loading" : text);
-    return '<span class="' + className + '">' + icon('spinner') + string + '...</span>';
+    let string = translate(text === null ? 'loading_ellipsis' : text);
+    return '<span class="' + className + '">' + icon('spinner') + string + '</span>';
```

Comment:
Note that I removed the hard-coded "..." here in addition to changing the default string. The only override used in the code is `relais_loading`, which already includes ellipses -- so this is actually a bug fix, removing redundant punctuation.